src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/DataSetVersionServiceTests.cs

```diff
@@ -250,4 +255,78 @@ await service.UpdateVersionsForReleaseVersion(
             Assert.Equal(releaseVersion2.Release.Title, actualDataSetVersion2.Release.Title);
         }
     }
+
+    public class UpdateStatusToFinalizingTests(TestApplicationFactory testApp)
```

Comment:
I didn't review this, because I think that updating the status in the Admin is the wrong place to be doing it. See my comment in DataSetVersionService.cs.

Reply:
Cool 🙂 

Just check the imports in the diff of files changed on this PR, because there's 5 unused imports being added to this file which no longer need to be there after the rest of the changes to it were reverted.